mkosi/resources/man/mkosi.1.md

```diff
@@ -204,80 +204,67 @@ The following command line verbs are known:
 
 ## Commandline-only Options
 
-Those settings cannot be configured in the configuration files.
+The following settings cannot be configured in the configuration files:
+
+`--help`, `-h`
+:   Show brief usage information.
```

Comment:
Moving it away from the middle makes sense, but first is too prominent and nobody looking at the manual will look there for this. Better move it to the end, just like in the actual usage info.